src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -31,12 +31,24 @@ const fieldNames = {
   emailsAmountByInterval: 'emailsAmountByInterval',
   intervalInDays: 'intervalInDays',
   excludedSubscribersLists: 'excludedSubscribersLists',
+  timeRestrictionHourFrom: 'timeRestriction.hourFrom',
+  timeRestrictionHourTo: 'timeRestriction.hourTo',
+  timeRestrictionTimeSlotEnabled: 'timeRestriction.timeSlotEnabled',
+  timeRestrictionWeekdaysEnabled: 'timeRestriction.weekdaysEnabled',
 };
 
 export const ContactPolicy = InjectAppServices(
-  ({ dependencies: { dopplerUserApiClient, dopplerContactPolicyApiClient, appSessionRef } }) => {
+  ({
+    dependencies: {
+      dopplerUserApiClient,
+      dopplerContactPolicyApiClient,
+      experimentalFeatures,
+      appSessionRef,
+    },
+  }) => {
     const navigate = useNavigate();
     const [enabled, setEnabled] = useState(false);
+    const [timeRestrictionEnabled, setTimeRestrictionEnabled] = useState(false);
```

Comment:
Since `timeRestrictionEnabled` is static (localStorage is only read on the first rendering), I think that you can simplify it in this way:

```suggestion
    // TODO: remove once the whole "time restriction" feature is implemented
    const timeRestrictionEnabled = experimentalFeatures.getFeature('timeRestrictionEnabled');
```

And remove the code in the _useEffect block_.

Reply:
Done: https://github.com/FromDoppler/doppler-webapp/pull/2385/commits/4f9b8b0f61faba08b267cf68b5c258bf43e9af8f